files/ROV_firmware_software_versions.md

```diff
@@ -0,0 +1,32 @@
+# ROV _Nereo_ firmware & software 
+<table>
+
+<tr><td> <i>item</i> </td><td> <i>type</i> </td><td> <i>version</i> </td><td> <i>commit</i> </td><td> <i>notes</i> </td></tr>
+
+<tr><td> ArduSub </td><td> ROV firmware </td><td> 4.5.0 (BETA) </td><td>  </td><td>  </td></tr>
+
+<tr><td> BlueOS </td><td> ROV software </td><td> 1.3.0 </td><td> cc71b67e </td><td>  </td></tr>
+
+<tr><td> QGC </td><td> flight control interface </td><td> 4.4.1 </td><td>  </td><td>  </td></tr>
+
+<tr><td> WL UGPS External </td><td> BlueOS Extension </td><td> v1.0.3 </td><td>  </td><td>  </td></tr>
+
+<tr><td> Water Linked UGPS </td><td> BlueOS Extension </td><td> v1.0.6 </td><td>  </td><td> </td></tr>
+
+<tr><td> Water Linked DVL </td><td> BlueOS Extension </td><td> v1.0.7 </td><td>  </td><td>  </td></tr>
+
+<tr><td> surftrak_fixit </td><td> BlueOS Extension </td><td> v1.0.0-beta.2 </td><td>  </td><td> </td></tr>
+
+<tr><td> major_tom </td><td> BlueOS Extension </td><td> 2024-07-29 </td><td>  </td><td> public.ecr.aws/blueos/bcloud-agent </td></tr> 
```

Comment:
@clydemcqueen do you know what the BlueOS Extension `major_tom` is? It's installed on Nereo, but I can't remember anything about this extension.

Reply:
This came up on the BlueOS Discord server, it appears to be a service that  uploads to / downloads from a new BR cloud service. It is in beta test, so (1) no need to list it in this doc, (2) you can / should disable it.

Reply:
Got it, thank you! I'll remove it from here and from Nereo.